src/sql/src/plan/statement/ddl.rs

```diff
@@ -1725,6 +1725,79 @@ pub fn plan_create_views(
     }
 }
 
+pub fn describe_create_recorded_view(
+    _: &StatementContext,
+    _: CreateRecordedViewStatement<Aug>,
+) -> Result<StatementDesc, anyhow::Error> {
+    Ok(StatementDesc::new(None))
+}
+
+pub fn plan_create_recorded_view(
```

Comment:
A lot of this method looks like it's duplicated from `plan_view`. It would be nice to find a way to consolidate the logic, but not a huge deal either way.